.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,5 +3,29 @@
 _Please add a description of the changes that this PR introduces and the files that
 are the most critical to review._ 
 
-Closes: #XXX
+If this PR is non-trivial/large/complex, please ensure that you have either
+created an issue that the team's had a chance to respond to, or had some
+discussion with the team prior to submitting substantial pull requests. The team
+can be reached via GitHub Discussions or the Cosmos Network Discord server in
+the #cometbft channel. GitHub Discussions is preferred over Discord as it
+allows us to keep track of conversations topically.
+https://github.com/cometbft/cometbft/discussions
+
+If the work in this PR is not aligned with the team's current priorities, please
+be advised that it may take some time before it is merged - especially if it has
+not yet been discussed with the team.
+
+See the project board for the team's current priorities:
+https://github.com/orgs/cometbft/projects/1
+
+-->
+
+---
+
+#### PR checklist
+
+- [ ] Tests written/updated
+- [ ] Changelog entry added in `.changelog` (we use
+  [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
```

Comment:
we can start using this if we want, but we might want to remove this if not

Reply:
Yeah we don't have to but it might be annoying if we have two means of reporting changes within the same repo

Reply:
I guess we could just continue to point to the cometbft changelog as well